### PR TITLE
feat(SU): MVP SU creation

### PR DESCRIPTION
**Pull request by WikiRik**

### CODE REVIEW

**Comment by WikiRik** [comment]
@serge1peshcoff I removed the credentials from src/credentials.js btw. Do you think it will break the map on prod?

**Comment by serge1peshcoff** [comment]
yeah, it will. You can leave it there as this token should only work for my.aegee.eu (you can specify the domain in settings)

File: src/services.json [hunk 1]
```diff
@@ -2,10 +2,12 @@
   "core": "/api/core",
   "mailer": "/api/mailer",
   "events": "/api/events",
+  "summeruniversity": "/api/summeruniversity",
```
Comment (serge1peshcoff) [inline src/services.json:5]: can we go with something shorter? like /api/su?

Comment (WikiRik) [reply src/services.json:5]: It felt too short for me, but summeruniversity is also a bit too long. We can change it later as well

**Comment by WikiRik** [comment]
> yeah, it will. You can leave it there as this token should only work for my.aegee.eu (you can specify the domain in settings)

Okay, I'll put it back

**Pull request merged**